src/Comments.php

```diff
@@ -17,16 +17,13 @@
  *
  * If the resulting yaml file contents are reordered, comments may
  * become mis-ordered (attached to the wrong element).
- *
- * Comments that appear before sections of yaml that are edited may
- * be inadvertantly lost. It is recommended to always place comments
- * immediately before identifier lines (i.e. "foo:").
```

Comment:
I don't think that this enhancement entirely obviates the risk described in this comment. I think we should bring this warning back.

Reply:
PR has been updated